src/middlewares/question/isQuestionPresent.middleware.ts

```diff
@@ -7,9 +7,16 @@ import { questionService } from '../../services';
 
 export const isQuestionPresentMiddleware = async (req: IRequestExtended, res: Response, next: NextFunction) => {
   try {
-    const { question_id } = req.params;
-    const question = await questionService.getQuestionById(question_id);
+    const {question_id} = req.params;
+    if (!question_id) {
```

Comment:
Will be better to use native mongo function
https://stackoverflow.com/questions/13850819/can-i-determine-if-a-string-is-a-mongodb-objectid

Reply:
Done